src/api/teams.spec.ts

```diff
@@ -0,0 +1,44 @@
+import request from 'supertest';
+import app from 'app';
+
+describe("teams", () => {
+    it('sanity test', () => {
+        expect(true).toBe(true);
+    })
+
+    it("should respond with a 200 status code", async () => {
+        const response = await request(app).get("/teams");
+        expect(response.statusCode).toBe(200);
+    })
+
+    it("should get a teams by its email and name", async () => {
```

Comment:
Team in singular, not plural: "should get a team".

Reply:
done.